src/bot/commands/settings.command.ts

```diff
@@ -156,19 +231,19 @@ export class SettingsCommand {
           }
         }
       }
-    }
 
-    if (dto.clear_intro_song) {
-      await this.virgins.nativeUpdate(
-        { id: target_user_snowflake },
-        { intro_song: null },
-      );
-
-      messages.push(
-        `${pascal_spaces(
-          target_user_name,
-        )} intro song has been reset to default.`,
-      );
+      if (dto.clear_intro_song) {
+        await this.virgins.nativeUpdate(
+          { id: target_user_snowflake },
+          { intro_song: null },
+        );
+
+        messages.push(
+          `${pascal_spaces(
+            target_user_name,
+          )} intro song has been reset to default.`,
```

Comment:
should this be capitalize 'intro' ?

Reply:
This turns in to "Your intro song has been reset to default."